### PR TITLE
tiup-playground: add info about TiProxy

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v9.0 (TiDB 9.0 versions)
- [x] v8.5 (TiDB 8.5 versions)
- [ ] v8.4 (TiDB 8.4 versions)
- [ ] v8.3 (TiDB 8.3 versions)
- [ ] v8.2 (TiDB 8.2 versions)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tiup/tiup-playground.md [hunk 1]
```diff
@@ -151,6 +151,28 @@ tiup playground scale-out --db 2
...
 
+## 部署 TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。你可以通过 TiUP Playground 为集群部署 TiProxy。
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
+命令行参数说明：
```
Comment (qiancai) [inline tiup/tiup-playground.md:162]:
```suggestion
tiup playground 命令行中与 TiProxy 相关的说明：
```

File: tiup/tiup-playground.md [hunk 2]
```diff
@@ -151,6 +151,28 @@ tiup playground scale-out --db 2
 tiup playground scale-in --pid 86526
 ```
 
+## 部署 TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。你可以通过 TiUP Playground 为集群部署 TiProxy。
```
Comment (qiancai) [inline tiup/tiup-playground.md:156]:
```suggestion
[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。

你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
```

File: tiup/tiup-playground.md [hunk 3]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
+
+你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
+`tiup playground` 命令行中与 TiProxy 相关的说明：
```
Comment (qiancai) [inline tiup/tiup-playground.md:164]:
```suggestion
`tiup playground` 命令行中与 TiProxy 相关的选项说明：
```

File: tiup/tiup-playground.md [hunk 4]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
+`tiup playground` 命令行中与 TiProxy 相关的选项说明：
+
+```bash
+Flags:
+      --tiproxy int                设置集群中 TiProxy 节点的数量
```
Comment (djshow832) [inline tiup/tiup-playground.md:168]:
```suggestion
      --tiproxy int                设置集群中 TiProxy 节点的数量。如果没有提供，不会部署 TiProxy。
```

File: tiup/tiup-playground.md [hunk 5]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
 tiup playground scale-in --pid 86526
 ```
 
+## 部署 TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，它放置在客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
```
Comment (lilin90) [inline tiup/tiup-playground.md:156]:
```suggestion
[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，位于客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
```

File: tiup/tiup-playground.md [hunk 6]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+`tiup playground` 命令行中与 TiProxy 相关的选项说明：
+
+```bash
+Flags:
+      --tiproxy int                设置集群中 TiProxy 节点的数量。如果未指定，不会部署 TiProxy。
+      --tiproxy.binpath string     指定 TiProxy 的二进制文件位置
+      --tiproxy.config string      指定 TiProxy 的配置文件
+      --tiproxy.host host          Playground 的 TiProxy host。如果没有提供，TiProxy 会使用 host 参数作为它的 host
+      --tiproxy.port int           Playground 的 TiProxy 端口。如果没有提供，TiProxy 会使用 6000 作为它的端口
+      --tiproxy.timeout int        TiProxy 最长等待超时时间，单位为秒，若配置为 0，则永不超时（默认为 60）。
```
Comment (lilin90) [inline tiup/tiup-playground.md:173]:
```suggestion
      --tiproxy int                设置集群中 TiProxy 节点的数量。如果未指定，不会部署 TiProxy。
      --tiproxy.binpath string     指定 TiProxy 的二进制文件位置。
      --tiproxy.config string      指定 TiProxy 的配置文件。
      --tiproxy.host host          Playground 的 TiProxy host。如果没有提供，TiProxy 会使用 host 参数作为它的 host。
      --tiproxy.port int           Playground 的 TiProxy 端口。如果没有提供，TiProxy 会使用 6000 作为它的端口。
      --tiproxy.timeout int        TiProxy 最长等待超时时间，单位为秒。若配置为 0，则永不超时（默认为 60）。
```

File: tiup/tiup-playground.md [hunk 7]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+Flags:
+      --tiproxy int                设置集群中 TiProxy 节点的数量。如果未指定，不会部署 TiProxy。
+      --tiproxy.binpath string     指定 TiProxy 的二进制文件位置。
+      --tiproxy.config string      指定 TiProxy 的配置文件。
+      --tiproxy.host host          Playground 的 TiProxy host。如果没有提供，TiProxy 会使用 host 参数作为它的 host。
+      --tiproxy.port int           Playground 的 TiProxy 端口。如果没有提供，TiProxy 会使用 6000 作为它的端口。
+      --tiproxy.timeout int        TiProxy 最长等待超时时间，单位为秒。若配置为 0，则永不超时（默认为 60）。
+```
+
+从 TiUP v1.14.1 版本起，你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
```
Comment (qiancai) [inline tiup/tiup-playground.md:170]:
```suggestion
从 TiUP v1.14.1 起，你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
```

**Review by djshow832 — APPROVED** [review]

**Review by lilin90 — APPROVED** [review]
/lgtm

File: tiup/tiup-playground.md [hunk 8]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
 tiup playground scale-in --pid 86526
 ```
 
+## 部署 TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，位于客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
+
+从 TiUP v1.14.1 版本起，你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
```
Comment (djshow832) [inline tiup/tiup-playground.md:158]: 从 TiUP v1.14.1 版本起，你可以通过 TiUP Playground 为集群部署 TiProxy。

1. 配置 tidb.toml：
```
graceful-wait-before-shutdown=15
```

2. 启动 TiDB 集群：

File: tiup/tiup-playground.md [hunk 9]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
+```
+
```
Comment (djshow832) [inline tiup/tiup-playground.md:163]: 如果要使用 tiproxyctl，详情参考[安装 TiProxy Control](/tiproxy-command-line-flags.md#安装-tiproxy-control)。

File: tiup/tiup-playground.md [hunk 10]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
 ```
 
+## 部署 TiProxy
+
+[TiProxy](/tiproxy/tiproxy-overview.md) 是 PingCAP 的官方代理组件，位于客户端和 TiDB server 之间，为 TiDB 提供负载均衡、连接保持、服务发现等功能。
+
+从 TiUP v1.14.1 版本起，你可以通过 TiUP Playground 为集群部署 TiProxy。例如：
+
+```shell
+tiup playground v8.5.0 --tiproxy 1
```
Comment (djshow832) [inline tiup/tiup-playground.md:161]:
```suggestion
tiup playground v8.5.0 --tiproxy 1 --db.config tidb.toml
```

File: tiup/tiup-playground.md [hunk 11]
```diff
@@ -151,6 +151,30 @@ tiup playground scale-out --db 2
...
+
+```bash
+Flags:
+      --tiproxy int                设置集群中 TiProxy 节点的数量。如果未指定，不会部署 TiProxy。
+      --tiproxy.binpath string     指定 TiProxy 的二进制文件位置。
+      --tiproxy.config string      指定 TiProxy 的配置文件。
+      --tiproxy.host host          Playground 的 TiProxy host。如果没有提供，TiProxy 会使用 host 参数作为它的 host。
+      --tiproxy.port int           Playground 的 TiProxy 端口。如果没有提供，TiProxy 会使用 6000 作为它的端口。
+      --tiproxy.timeout int        TiProxy 最长等待超时时间，单位为秒。若配置为 0，则永不超时（默认为 60）。
+```
```
Comment (djshow832) [inline tiup/tiup-playground.md:176]: --tiproxy.version string         指定 TiProxy 的版本号。如果没有提供，会部署最新的 TiProxy 版本。

**Pull request merged**